content/en/docs/Core tasks/install.md

```diff
@@ -103,6 +103,10 @@ kubectl get events -n cf-operator --watch
 
 The main input to the operator is the `BOSH deployment` custom resource and the according manifest config map or secret. Changes to the `Spec` or `Data` fields of either of those will trigger the operator to recalculate the desired state and apply the required changes from the current state.
 
+> **Note:**
+>
+> Each creation or update can trigger the creation of multiple pods with the `dm-` or `ig-` prefix, those are _warm up_ jobs which are preparating the pods where your BOSH releases will be running to. When `dm-` and `ig-` pods are completed, you should have your deployment ready and running. See also [Bosh deployment status](../../development/controllers/bosh_deployment/#boshdeployment-status).
```

Comment:
```suggestion
> Each creation or update can trigger the creation of multiple pods with the `dm-` or `ig-` prefix. Those are _warm up_ jobs, which are preparing manifests and eventually configure the pods your BOSH releases will be running on. When `dm-` and `ig-` pods are completed, the pods are ready to start up. See also [Bosh deployment status](../../development/controllers/bosh_deployment/#boshdeployment-status).
```

Reply:
Thank you :+1:  was getting messy  :spaghetti:  spaghetti indeed!